types/office-js-preview/index.d.ts

```diff
@@ -21402,7 +21402,7 @@ declare namespace Office {
         /**
          * Gets the account type of the user associated with the mailbox.
          *
-         * **Note**: This member is currently only supported in Outlook on Mac, Build 16.9.1212 and greater.
+         * **Note**: This member is currently only supported in Outlook on Mac starting in Version 16.9 (18121200).
```

Comment:
since it was 16.9.1212, do we need to go up a version from the released 16.9.1?

```suggestion
         * **Note**: This member is currently only supported in Outlook on Mac starting in Version 16.10.0 (18021001).
```